README.md

```diff
@@ -1,833 +1,477 @@
 
-The role IDs are passed as part of the `options` in `generateBearerToken(filepath, options)` function, which takes the service account credentials file path and an optional `options` object for token generation. 
+#### Example: Upsert Records (update or insert)
 
-[Example using a service account credentials file path:](https://github.com/skyflowapi/skyflow-node/blob/master/samples/service-account/ScopedTokenGenerationExample.ts)
+Insert call [example](https://github.com/skyflowapi/skyflow-node/blob/master/samples/vault-api/UpsertSupport.ts) with upsert support:
 
-```js
-import { generateBearerToken, isExpired } from 'skyflow-node';
+```javascript
+const response = vault.insert({
+  // records to be inserted
+  records: [{
+    fields: {
+      expiry_date: '12/2026',
+      card_number: '411111111111111',
+    },
+    table: 'cards',
+  }],
+}, {
+  // optional options
+  tokens: true,
+  // To conditionally insert OR update based on a field
+  upsert: [
+    {
+      table: 'cards',         // Name of the table in the vault.
+      column: 'card_number',  // Name of the column in the vault. Must be defined as `unique`.
+    }
+  ]
+});
 
-let filepath = 'CREDENTIALS_FILE_PATH'
-let bearerToken = '';
+response.then(
+    (res) => {
+        console.log(JSON.stringify(res));
+    },
+    (err) => {
+        console.log(JSON.stringify(err));
+    }
+).catch((err) => {
+    console.log(JSON.stringify(err));
+});
+```
 
-function getSkyflowBearerToken() {
-    return new Promise(async (resolve, reject) => {
-        try {
-            const options = {
-                roleIDs: ['ROLE_ID1', 'ROLE_ID2'],
-            };
-            if(!isExpired(bearerToken)) resolve (bearerToken);
-            else {
-                let response = await generateBearerToken(filepath, options);
-                bearerToken = response.accessToken;
-                resolve(bearerToken);
-            }           
-        } catch (e) {
-            reject(e);
-        }
-    });
+Samples Response:
+```json
+{
+  "records": [
+    {
+      "table": "cards",
+      "fields": {
+        "skyflow_id": "16419435-aa63-4823-aae7-19c6a2d6a19f",
+        "card_number": "f3907186-e7e2-466f-91e5-48e12c2bcbc1",
+        "cvv": "1989cb56-63da-4482-a2df-1f74cd0dd1a5",
+      },
+    }
+  ]
 }
+```
 
-const tokens = async () => {
-    console.log(await getSkyflowBearerToken());
-};
+#### Example: Insert with partial success support
 
-tokens();
-```
-Alternatively, you can send the service account credentials as a string, by using the `generateBearerTokenFromCreds(credentials, options)` function.
-
-Note:
-By including context in the options, you can create scoped bearer tokens with the context JWT claim.
+<details>
+<summary>
+Insert with partial success support
+</summary>
```

Comment:
Why is this in an expandable but not the other examples? We should be consistent with example display/expandable usage.